views/index.ejs

```diff
@@ -401,6 +401,7 @@ const teneoProps = {
 
     'ttsActive': <%= process.env.TTS_ACTIVE || false %>,
     'asrActive': <%= process.env.ASR_ACTIVE || false %>,
+    'ratingSymbol':'<%= process.env.RATING_SYMBOL %>',
```

Comment:
Should this be defined and explained in app.json?

Reply:
done